web/html/src/manager/index.js

```diff
@@ -41,6 +41,7 @@ const pages = {
   'virtualization/guests/list/guests-list.renderer': './manager/virtualization/guests/list/guests-list.renderer.js',
   'virtualization/guests/edit/guests-edit.renderer': './manager/virtualization/guests/edit/guests-edit.renderer.js',
   'activation-key/activation-key-channels.renderer': './manager/activation-key/activation-key-channels.renderer.js',
+  'virtualization/guests/create/guests-create.renderer': './manager/virtualization/guests/create/guests-create.renderer.js',
```

Comment:
Due to this [PR](https://github.com/uyuni-project/uyuni/pull/506) you can now define your routes like this: https://github.com/lneves12/uyuni/blob/5c04dac65ec9c9c7e6e8c4bfd92b4d14bc05d315/web/html/src/manager/content-management/index.js

It's not needed for this PR, but would be nice to eventually migrate all the virtualization frontend routes into this new format